bundle/manifests/lvm-operator.clusterserviceversion.yaml

```diff
@@ -28,11 +28,38 @@ metadata:
       ]
     capabilities: Basic Install
     categories: Storage
+    containerImage: quay.io/ocs-dev/lvm-operator:latest
+    description: The ODF LVM Operator manages local storage using LVM
+    operatorframework.io/cluster-monitoring: "true"
+    operatorframework.io/initialization-resource: |-
+      {
+          "apiVersion": "lvm.topolvm.io/v1alpha1",
+          "kind": "LVMCluster",
+          "metadata": {
+            "name": "odf-lvmcluster"
+          },
+          "spec": {
+            "storage": {
+              "deviceClasses": [
+                {
+                  "name": "vg1",
+                  "thinPoolConfig": {
+                    "name": "thin-pool-1",
+                    "overprovisionRatio": 10,
+                    "sizePercent": 90
+                  }
+                }
+              ]
+            }
+          }
+        }
     operatorframework.io/suggested-namespace: openshift-storage
     operators.operatorframework.io/builder: operator-sdk-v1.13.0+git
     operators.operatorframework.io/internal-objects: '["logicalvolumes.topolvm.cybozu.com",
       "lvmvolumegroups.lvm.topolvm.io", "lvmvolumegroupnodestatuses.lvm.topolvm.io"]'
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+    repository: https://github.com/red-hat-storage/lvm-operator
+    support: Red Hat
```

Comment:
Please remove this line as the LVMO is not officially supported yet.

Reply:
done.